utils/__init__.py

```diff
@@ -0,0 +1,13 @@
+from random import randint
+
+class Colors:
+    def __init__(self):
+        self.color_list = [0x36a64f, 0x288BA8, 0x746AB0, 0xE83845, 0xFFCE30, 0xE389B9]
+
+    def random_color(self):
```

Comment:
Not essential, so feel free to ignore this, but do we actually want random color? I'm thinking about using color for job platforms (e.g LinkedIn vs Glassdoor, etc), or for company level (e.g. Netflix vs a random company).
We can talk more about this later. But I think it's still nice to use this random feature now.